docs/cow-protocol/tutorials/arbitrate/README.mdx

```diff
@@ -133,3 +133,125 @@ sequenceDiagram
     Note over autopilot: End
 ```
 
+## Run locally end to end
+
+In this tutorial we will run all the components locally on Ethereum mainnet, including the orderbook, autopilot, driver and solver engine. All these components have their own configuration parameters. For the sake of simplicity, we will use the bare minimum configuration for all of them.
+
+## Solver engine configuration file
+
+baseline.toml
```

Comment:
Thief! 😄 
To make this easier to run we can also consider writing the config files inside `run.sh`. Then the user only has to copy and paste a single file.
Not sure what is nicer for a tutorial, though.

Reply:
![gollum](https://github.com/cowprotocol/docs-v2/assets/19595624/870cfb70-f5df-46d2-8732-11b36e407e28)

Reply:
![cat](https://github.com/cowprotocol/docs-v2/assets/19595624/f5e50fda-b928-4711-a524-37f6f6ae99ab)